pkg/kubelet/dockertools/manager_test.go

```diff
@@ -1396,7 +1398,7 @@ func TestSyncPodBackoff(t *testing.T) {
 		fakeDocker.ContainerList = containerList
 		fakeClock.Time = startTime.Add(time.Duration(c.tick) * time.Second)
 
-		runSyncPod(t, dm, fakeDocker, pod, backOff)
+		runSyncPod(t, dm, fakeDocker, pod, backOff, reflect.DeepEqual(c.result, backOffCalls))
```

Comment:
(nit) I prefer to make expectations explicit in tests. I'd add an `expectError bool` to the tests struct.